scripts/manage-db.py

```diff
@@ -13,22 +14,74 @@
 
 from auslib.db import AUSDatabase
 
+def cleanup_releases(trans, nightly_age, dryrun=True):
+    # This and the subsequent queries use "%%%%%" because we end up going
+    # through two levels of Python string formatting. The first is here,
+    # and the second happens at a low level of SQLAlchemy when the transaction
+    # is being executed.
+    query = """
+LEFT OUTER JOIN releases_history USING (name)
+LEFT JOIN rules ON (name=mapping)
```

Comment:
re: LEFT JOIN vs. LEFT OUTER JOIN - I don't think there's any difference either, I just didn't tweak those parts of the queries from https://bugzilla.mozilla.org/show_bug.cgi?id=1237991#c41.

It's a good point about only needing the most recent row - I wonder if it would be faster to do subquery with LIMIT 1 instead of a JOIN?